src/.nuget/NuGet.Config

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file was intentionally deleted in 1ea71faaa6d71d88e37b3f98ff93f379b8707636. I think it'd be better to update the solution item to point to the new file in the root of the repo.